python/src/iceberg/schema.py

```diff
@@ -40,6 +41,8 @@
 
 T = TypeVar("T")
 
+logger = logging.getLogger(__name__)
```

Comment:
Looks like this is unused. Do we need to add it?

Reply:
No, let's remove it!